src/common/settings.js

```diff
@@ -452,6 +452,12 @@ const character_settings = {
         "description": "Unleash your divine soul to deal extra radiant damage equal to your level.",
         "type": "bool",
         "default": false
+    },
+    "wizard-bladesong": {
+        "title": "Bladesong: Your magic causes your weapon to sing",
```

Comment:
Format for every other special feat is `${class}: ${feature name}`, so I would do `"Wizard: Bladesong"` here instead, and in the description say something like "Activate your Bladesong and make your weapon sing with magic" or something like that. No need for the details of the mechanics

Reply:
Fixed in PR update that is incoming.